src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,64 @@
+import classnames from 'classnames';
+import { Person } from '../types/Person';
+
+type Props = {
+  searchResults: Person[] | null;
+  onSelect: (person: Person) => void;
+  setIsDropdownActive: (isActive: boolean) => void;
+  activePersonSlug: string | null;
+
+};
+
+export const DropdownMenu: React.FC<Props> = (
+  {
+    searchResults,
+    onSelect,
+    setIsDropdownActive,
+    activePersonSlug,
+  },
+) => {
+  if (!searchResults) {
+    return (
+      <div className="dropdown-menu" role="menu">
+        <div className="dropdown-content">
+          <div className="dropdown-item">
+            <p
+              className="has-text-danger"
+            >
+              No matching suggestions
+            </p>
+          </div>
+        </div>
+      </div>
+    );
+  }
```

Comment:
Since you now check for searchResults.length === 0 in line 20, you can remove this condition from line 39. Just render the searchResults.map() inside the 'dropdown-content' div.